li-hadoop-plugin/src/main/groovy/com/linkedin/gradle/libangbang/LiPigCommand.groovy

```diff
@@ -19,8 +19,6 @@ package com.linkedin.gradle.libangbang;
 
```

Comment:
Could you also clean up the extraneous whitespace lines at the top of this file?